lib/core_extensions.rb

```diff
@@ -140,17 +140,20 @@ def to_translation
   end
 
   def to_gb
-    value, _, unit=self.match(/(\d+(\.\d+)?) ?(([KMGT]i?B?|B))$/i)[1..3]
+    match_data = self.match(/(\d+(\.\d+)?) ?(([KMGT]i?B?|B|Bytes))$/i)
```

Comment:
I didn't want to mix refactorings with fix since I thought this should be merged into 1.13 but since it was rejected I can fix existing code.